One create seeder for series

SebastianGerS opened the pull request:
Both seeders are working now!
I also made some changes to the photos_table to make it possible to link a photo to either a title or a photo (also made some changes to the respective models for this change to work).

The seeders them selves are a bit messy.. I whant to refactor them eventualy, but perhaps thats not the most important thing right now?

or should I do that before we merge this?

Commit:
chang function names to camlecase

Commit:
change to corect functionnames in seeder

Commit:
create test controler and some functionality

	* create test controller
	* add func for adding series and seasons to databse from
	  externalapi
	* parshaly create functinality for adding episodes from external
	  api to db

Commit:
add timstamps = false to series, episodes and seasons

	this fixes default laravel setting that demands timestaps when
	inserting directly into tables

Commit:
change column properties

	* change pg_rating in series_table to nullable()
	* change plot_summary in episode_table to nullable()

	thees changes where necesary becous the apropriate data where not
	always found in the database used to seed thees tables

Commit:
Fix correct function name in seeder

	* changed function name producerOTitle to producerOfTitle

Commit:
Fixed functionality for seeder

	* fix suport for seeding episodes
	* change requests to only go to themoviedb
	* add suport for seeding producername for entire series
	* add suport for seeding serie creator
	* add suport for calling SerieSeederwhen seeding the database

Commit:
add more series and movies

Commit:
fix make pg_rating column nullable

Commit:
fix remove calls to theimdbapi

	* remove calĺs to theimdbapi
	* replace data previusly deliverd from imdbapi with data from
	  themoviedb

	  themoviedb is much more reliable when in comes to conections,
	  (keept getting bad getway error_message from imdbapi) the downside is 	  that we cant pg_ratings for movies and will have to be filld in
	  manualy from now on.

Commit:
Add suport for linking photos to persons

	* change function name in Photo Model to imageable()
	* change call in imageable to morphTo
	* change call in photos() in Title Model to morphMany()
	* add function photos with call to morphMany() in Person Model
	* add morph column to photos_table and removed title_id column
	* add width and ratio column to photos_table

Commit:
Update Seeders to suport changes in photos_table

	* change call to create new photo for title in photos_table to
	  also contain data about width, ratio, imagebletype, and
	  photo_type
	* add call when fetching new person to add profile pictures of
	  that person
	* add suport for fetching posters and backdrops for each episode

Commit:
Modify and renamed files and variablenames

	* Delete the serieseedertester
	* Rename sedeers
	* Update calls to corect classes in DatabaseSedeer due to
	  namechange
	* Rename all varibles with _ to insted use camleCase
	* Fix row spacing for increased readability
	* Add func for adding profile photos of crew in movies

Commit:
minor cleenup

	* removed route to previusly removed seriessedeertester
	* add append images to call when getting crew members

ghost commented:
Looks good

Den 17 dec. 2017 20:43 skrev "SebastianGerS" <notifications@github.com>:

> @SebastianGerS <https://github.com/sebastiangers> requested your review
> on: chas-academy/berserkers-06-imdb-clone#55
> <https://github.com/chas-academy/berserkers-06-imdb-clone/pull/55> One
> create seeder for series.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/chas-academy/berserkers-06-imdb-clone/pull/55#event-1391183029>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AbMc3DDkyBH9xYKibaTz3PZapFoY-aiNks5tBW7kgaJpZM4RExQ->
> .
>

Commit:
fix correct class name in DatabaseSeeder

Commit:
Add func to get pg_rating from another api

	* add new client conected to obdbapi
	* send request to client for each movie in the array
	* get rating property from omdbClinets response
	* modify loop over production_countries (this also fixes bug
	  where last country code got a ',' attached at the end

Commit:
temp change some movies and series commentedout for faster seeding
